test-e2e/members.js

```diff
@@ -116,6 +116,27 @@ test('getting yourself after adding project (but not yet synced)', async (t) =>
   )
 })
 
+test('cannot invite when project is missing critical details', async (t) => {
```

Comment:
noticed that this test was notably missing. not strictly necessary to add this as part of this PR, but figured it could be if we decide to make the project color a required field for invites as well.